# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/powerpc/linux24benh/96-quota-for-userspace.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

Samba needs the struct ...

 - Rene Rebe <rene@rocklinux.org>

--- linux-2.4.24/include/linux/quota.h.orig	2004-01-09 21:51:11.000000000 +0100
+++ linux-2.4.24/include/linux/quota.h	2004-01-09 21:55:10.000000000 +0100
@@ -132,12 +132,6 @@
 	__u32 dqi_valid;
 };
 
-#ifdef __KERNEL__
-
-#include <linux/dqblk_xfs.h>
-#include <linux/dqblk_v1.h>
-#include <linux/dqblk_v2.h>
-
 /*
  * Data for one user/group kept in memory
  */
@@ -152,6 +146,12 @@
 	time_t dqb_itime;	/* time limit for excessive inode use */
 };
 
+#ifdef __KERNEL__
+
+#include <linux/dqblk_xfs.h>
+#include <linux/dqblk_v1.h>
+#include <linux/dqblk_v2.h>
+
 /*
  * Data for one quotafile kept in memory
  */
